lib/search.rb

```diff
@@ -1191,6 +1199,10 @@ def posts_eager_loads(query)
       topic_eager_loads << :tags
     end
 
+    Search.custom_topic_eager_loads.each do |custom_loads|
+      topic_eager_loads = custom_loads.is_a?(Array) ? custom_loads : custom_loads.call(search_pms: @search_pms).to_a
```

Comment:
Sorry this doesn't look right. Shouldn't we be pushing or concating into the array instead of overriding it?